src/content/self-hosted/aks.mdx

```diff
@@ -41,6 +41,8 @@ Run the following command to obtain your cluster's API server endpoint:
 kubectl config view --minify | grep server
 ```
 
+> Our installation guides assume Okteto will be running in a dedicated cluster. If you want to install Okteto in a cluster with other workloads, we recomment to contact support for extra guidance.
```

Comment:
This is a thought . I don't think the "with other workloads." phrase is clear. 

```suggestion
> Our installation guides assume Okteto will be running in a dedicated cluster. We recommend [contacting our team](https://www.okteto.com/contact/)  if you plan on installing Okteto in a cluster with other workloads.
```

Reply:
This reads well to me. /contact routes to sales and they will need to pass over to support, but I like sending them to that form so they can fill out with more context.